x-pack/plugins/infra/public/components/logging/log_text_wrap_controls.tsx

```diff
@@ -5,25 +5,41 @@
  */
 
 import { EuiFormRow, EuiSwitch } from '@elastic/eui';
+import { InjectedIntl, injectI18n } from '@kbn/i18n/react';
 import * as React from 'react';
 
 interface LogTextWrapControlsProps {
   wrap: boolean;
   setTextWrap: (scale: boolean) => any;
+  intl: InjectedIntl;
 }
 
-export class LogTextWrapControls extends React.PureComponent<LogTextWrapControlsProps> {
+class LogTextWrapControlsUI extends React.PureComponent<LogTextWrapControlsProps> {
   public toggleWrap = () => {
     this.props.setTextWrap(!this.props.wrap);
   };
 
   public render() {
-    const { wrap } = this.props;
+    const { wrap, intl } = this.props;
 
     return (
-      <EuiFormRow label="Line Wrapping">
-        <EuiSwitch label="Wrap long lines" checked={wrap} onChange={this.toggleWrap} />
+      <EuiFormRow
+        label={intl.formatMessage({
+          id: 'xpack.infra.logs.customizeLogs.lineWrappingFormRowLabel',
+          defaultMessage: 'Line Wrapping',
+        })}
+      >
+        <EuiSwitch
+          label={intl.formatMessage({
+            id: 'xpack.infra.logs.customizeLogs.wrapLongLinesFormRowLabel',
```

Comment:
Wouldn't this be more like `xpack.infra.logs.customizeLogs.wrapLongLinesSwitchLabel`?

Reply:
Change it